### ISSUE TITLE
Search Method

### ISSUE THREAD

**Issue by zhoufanyi** [zhoufanyi/-507F17_Final_Project#11]
- [ ] Search Method to find players stats and team stats.

**Comment by aerenchyma** [comment]
I'm not sure what you mean by search method here -- does this reference a python function to make queries to the database?

**Comment by zhoufanyi** [comment]
Yes, I want to make queries to the database with python code. For example, select stats of all small forwards  and get the standings of all west team etc. . Maybe, I want to get stats of a specific player.
